.travis.yml

```diff
@@ -26,10 +26,7 @@ install:
   - python setup.py -q develop
 
 script:
-  - |
-      openwisp-utils-qa-checks \
-        --migration-path ./django_netjsonconfig/migrations/ \
-        --migration-module django_netjsonconfig
+  - ./run-qa-checks
```

Comment:
move to this script also `jslint django_netjsonconfig/static/django-netjsonconfig/js/*.js` from above please

Reply:
done!